src/FormControls/RadioButton.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default class RadioButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getInputDOMNode() {
+    return React.findDOMNode(this.refs.input);
+  }
+
+  getValue() {
+    return this.props.value;
+  }
+
+  getChecked() {
+    return this.getInputDOMNode().checked;
+  }
+
+  renderLabel(input) {
+    if (!this.props.label) {
+      return input;
+    }
+
+    const {id, label} = this.props;
+
+    const classes = {
+      'radio': !this.props.inline,
+      'radio-inline': this.props.inline
+    };
+
+    return (
+      <label className={classNames(classes, this.props.labelClassName)} htmlFor={id}>
```

Comment:
This is where my headache's been so far. For regular `radio` buttons the styling works when wrapped in the div as you said, but for inline it *must* be on the label without wrapping.